Practicum/Pract. 03/zad2.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <fstream>
+
+constexpr char INPUT_NAME[] = "Input.txt";
+constexpr char OUTPUT_NAME[] = "Output.txt";
+
+
+
+
+void copyOneFileToOther(std::ifstream& in,std::ofstream& out)
+{
+	char ch;
```

Comment:
you can copy at bigger chunks

Reply:
check the implementation here https://github.com/GeorgiTerziev02/Object-oriented_programming_FMI/blob/main/Sem.%2002/Solutions/CopyFileContent.cpp